src/Storefront/Resources/views/storefront/component/line-item/element/unit-price.html.twig

```diff
@@ -1,11 +1,17 @@
 {% block component_line_item_unit_price %}
+    {% set taxFootNote = "general.star"|trans|sw_sanitize %}
+
+    {% if not isTaxStateSame %}
+        {% set taxFootNote = ("general.star"|trans|sw_sanitize) ~ ("general.star"|trans|sw_sanitize) %}
```

Comment:
can you add some screenshots about the changes? i'm wondering where this double star footnote will be displayed or what do you want to accomplish here? :thinking:

Reply:
Added one and a bit of explanation. Is it helpful?

Reply:
yep definetly, but honestly i'll forward this to the PO in questions, as i'm not quite happy with the global and local foot notes :thinking:

Reply:
We would prefer if the foot note about the applied or not applied taxes is always shown in the card for the specific order and of course is based on the tax state of the order and not of the current state of the customer

the global foot notes then should be removed (emptied for now and the blocks deprecated)

What do you think?

Reply:
exactly

Reply:
Say no more!

Reply:
Done as new PR :) https://github.com/shopware/platform/pull/3115